src/CommunityToolkit.Maui.MediaElement/Services/MediaControlsService.android.cs

```diff
@@ -217,6 +217,15 @@ void OnSetIntents()
 		notification?.AddAction(actionNext);
 	}
 
+	public override void OnDestroy()
+	{
+		Platform.CurrentActivity?.StopService(new Intent(Platform.AppContext, typeof(MediaControlsService)));
+		System.Diagnostics.Trace.TraceInformation("MediaControlsService destroyed.");
+		Platform.CurrentActivity?.FinishAndRemoveTask();
+		System.Environment.Exit(0);
```

Comment:
are we sure that we want to call this method? This can lead on issues,since calling this method will terminate any ongoing process. Since this is inside the `OnDestroy` we shouldn't need to terminate anything

Reply:
This is about lifecycle behavior on restore after Android OS kills process while in background. If to save battery it kills the process the current behavior restores previous state upon launch and continues playback if you were playing on whatever page you app was on. The developer who made the bug report found this to be unexpected and inconsistent with what they expected. 

The code above will reset the life cycle of the app when you relaunch it and prevent the saved state from being restored. This is a way to prevent the state from being restored. It is listed in google docs as the expected way to tell Android not to save state. The lines:
```
Platform.CurrentActivity?.FinishAndRemoveTask();
System.Environment.Exit(0);
```
This is directly from Dotnet Maui repo for how they prevent the state from being restored automatically.

Reply:
i see, it's fine then, I just want to make sure about it